packages/react-core/src/components/Nav/NavExpandable.tsx

```diff
@@ -92,8 +95,13 @@ export class NavExpandable extends React.Component<NavExpandableProps, NavExpand
   render() {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const { id, title, srText, children, className, isActive, groupId, isExpanded, onExpand, ...props } = this.props;
+
     const { expandedState } = this.state;
 
+    const onClick = () => {
+      this.setState(prevState => ({ expandedState: !prevState.expandedState }));
```

Comment:
You need to call onClick here if it is provided.

Reply:
What do you mean?